File: app/src/main/java/com/ytempest/wanandroid/base/load/ViewType.java
package com.ytempest.wanandroid.base.load;

import androidx.annotation.IntDef;

import java.lang.annotation.Retention;
import java.lang.annotation.RetentionPolicy;

/**
 * @author heqidu
 * @since 2020/12/28
 */
@IntDef({ViewType.ERR, ViewType.LOAD,})
@Retention(RetentionPolicy.SOURCE)
public @interface ViewType {
    int ERR = 1;
    int LOAD = 2;
}

File: app/src/main/java/com/ytempest/wanandroid/base/load/ViewType.kt
package com.ytempest.wanandroid.base.load

import androidx.annotation.IntDef

/**
 * @author heqidu
 * @since 21-2-8
 */
@IntDef(ViewType.ERR, ViewType.LOAD)
@kotlin.annotation.Retention(AnnotationRetention.SOURCE)
annotation class ViewType {
    companion object {
        const val ERR = 1
        const val LOAD = 2
    }
}
